lti_consumer/plugin/compat.py

```diff
@@ -110,7 +110,10 @@ def get_course_by_id(course_key):
     Import and run `get_course_by_id` from LMS
     """
     # pylint: disable=import-error,import-outside-toplevel
-    from lms.djangoapps.courseware.courses import get_course_by_id as lms_get_course_by_id
+    try:
+        from openedx.core.lib.courses import get_course_by_id as lms_get_course_by_id
+    except ImportError:
+        from lms.djangoapps.courseware.courses import get_course_by_id as lms_get_course_by_id
```

Comment:
`Once the LMS has fully switched over to this new path, we can remove the legacy (LMS) import support here.` -> might add that as a comment or even a TODO in the code

Reply:
@sarina Done, TODO comment added to docstring.